pypeit/core/extract.py

```diff
@@ -1176,6 +1186,8 @@ def objfind(image, thismask, slit_left, slit_righ, inmask=None, fwhm=3.0, maxdev
             spectral direction smashed out).
         cont_fit (bool): default=True:
             Fit a continuum to the illumination pattern across the slit when peak finding
+        boxcar_rad_pix (float, optional): Boxcar radius; only for sky masking. Needs to be in pixels
+            If set, the skymask includes this boxcar radius
```

Comment:
change to sky mask is required to be at least as wide as this boxcar radius.

Reply:
edited